examples/globus_app.py

```diff
@@ -0,0 +1,155 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of Invenio.
+# Copyright (C) 2015, 2016, 2017 CERN.
+#
+# Invenio is free software; you can redistribute it
+# and/or modify it under the terms of the GNU General Public License as
+# published by the Free Software Foundation; either version 2 of the
+# License, or (at your option) any later version.
+#
+# Invenio is distributed in the hope that it will be
+# useful, but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Invenio; if not, write to the
+# Free Software Foundation, Inc., 59 Temple Place, Suite 330, Boston,
+# MA 02111-1307, USA.
+#
+# In applying this license, CERN does not
+# waive the privileges and immunities granted to it by virtue of its status
+# as an Intergovernmental Organization or submit itself to any jurisdiction.
+
+r"""Minimal Flask application example for development with globus handler.
+
+SPHINX-START
+
+1. Register a Globus application at `https://developers.globus.org/` with the
```

Comment:
Add reference to the [app registration instructions](https://docs.globus.org/api/auth/developer-guide/#register-app)

Reply:
There is a link on line 32, should it be more obvious?

Reply:
I think it's better to reference the app registration portion directly by adding the `#register-app` to the URL. More broadly, we should think about whether this is really just a developer activity. Deploying something like Invenio is a sys admin task.

Reply:
Good point, I added links to those specific sections